Sources/macOS/Extensions/BlueprintLayout+macOS.swift

```diff
@@ -45,6 +46,28 @@ extension BlueprintLayout {
     collectionView?.frame.size.height = contentSize.height
   }
 
+  /// When transitioning between layouts macOS does not set the
+  /// proper size to the document view of the scroll view.
+  /// To work around this, the collection views window will
+  /// temporarily be resized and then restored. This will
+  /// trigger the document view getting the new size.
+  func macOSWorkaroundSetNewContentSize() {
+    if let window = collectionView?.window {
+      CATransaction.begin()
+      CATransaction.setDisableActions(true)
+      CATransaction.setAnimationDuration(0.0)
+      let originalFrame = window.frame
+      var frame = originalFrame
+      frame.origin.y = frame.origin.y - 0.025
+      frame.size.height = frame.size.height + 0.025
+      window.setFrame(frame, display: false)
+      CATransaction.commit()
+      DispatchQueue.main.asyncAfter(deadline: .now() + 0.045) {
```

Comment:
The values used are not very scientific, it is just what I used that gained the best results when testing.